AppCenter/AppCenter/MSAppCenter.m

```diff
@@ -106,6 +106,15 @@ + (BOOL)isConfigured {
   return [MSAppCenter sharedInstance].sdkConfigured && [MSAppCenter sharedInstance].configuredFromApplication;
 }
 
++ (BOOL)isRunningInAppCenterTestCloud {
+    NSDictionary *environmentVariables = [[NSProcessInfo processInfo] environment];
+    NSString *runningInAppCenter = environmentVariables[kMSRunningInAppCenter];
+    if (runningInAppCenter == kMSTrue_Environment_String) {
```

Comment:
String comparison should be done by calling `isEqualToString:`.
Here is a [doc](https://developer.apple.com/documentation/foundation/nsstring/1407803-isequaltostring?language=objc).